variables.tf

```diff
@@ -9,6 +9,12 @@ variable "cloudflare_zone_name" {
   type        = string
 }
 
+variable "cloudflare_proxy_status" {
```

Comment:
Looks like the test failed because this variable isn't declared in the submodule. It should be in `modules/fail-over-cname/variables.tf` as well as here. It should also be added to the top-level module `main.tf` where the `fail-over-cname` module is instantiated (lines 27-36). (Here's a diagram of the module hierarchy: https://docs.google.com/drawings/d/1tBaqUQ01zAP2LWhuIJF2t6vUJgCuxkmB6rn3vi3ipf4/edit)